default/python/AI/ResearchAI.py

```diff
@@ -930,14 +908,13 @@ def generate_classic_research_orders():
                         if not tech_is_complete(_s_tech)
                     ]:
                         res = fo.issueEnqueueTechOrder(supply_tech, num_techs_accelerated)
-                        msg = (
-                            "Empire has poor colony/outpost prospects, so attempted to fast-track %s, got result %d"
-                            % (supply_tech, res)
+                        debug(
+                            (
+                                "Empire has poor colony/outpost prospects, so attempted to fast-track %s, got result %d",
+                                supply_tech,
+                                res,
+                            )
```

Comment:
I'm surprised that neither pycharm nor black handles that:
You could remove one pair of brackets here. Same in line 960 and 1145.